tls/server-tls-nonblocking.c

```diff
@@ -52,36 +52,34 @@ enum {
 
 static int tcp_select(SOCKET_T socketfd, int to_sec, int rx)
```

Comment:
Since we do not check the return of `tcp_select`, it would be better to change the return to a `void`. Alternatively, the return of this function should be checked in the `main` loop.

Reply:
Updated to poll() version. It looks like behaving as expected.

Will eliminate the return value as soon as debugging is completed.